Compass129Sources/COMfile/C12COM9.ASC

```diff
@@ -1363,21 +1363,21 @@ m_priortab	db	4,1,1,5,255,1,2,1,6,4,1,1,5,1
 	db	255,1,4,1,8,3,3,1,7,4,4,1,8,4,4,1
 	db	8,3,4,1,8,0
 
-inst	db	0	;0=nog een maagdelijk programma
-	;                        -->dus opstartlogo tonen+src_mem
-	db	0	;sla srcmem niet over
-i_new	equ	inst+#00	;splinternieuwe compass 0=ja
-i_skipsrcmem	equ	inst+#01	;moet de searchmem overgeslagen w?0=nee
+inst	db	0	;0=another virginal program
+	                        ;-->then show boot logo+src_mem
+	db	0	;don't skip srcem

```

Comment:
typo, missing 'm': srcmem (or perhaps make it src_mem for consistency with previous line?)